src/main/java/com/google/api/generator/engine/ast/ExprStatement.java

```diff
@@ -53,9 +53,11 @@ public ExprStatement build() {
                 || (expr instanceof ReferenceConstructorExpr)
                 || (expr instanceof AssignmentExpr)
                 || (expr instanceof ThrowExpr)
-                || (expr instanceof ReturnExpr),
+                || (expr instanceof ReturnExpr)
+                || (expr instanceof UnaryOperationExpr
+                    && ((UnaryOperationExpr) expr).isPostfixIncrement()),
```

Comment:
curious why unary operation expr has to be post-fix?

Reply:
`i++` is a valid standalone statement, but the second one in `boolean x = true; !x;` is not.

Reply:
I see. Thanks for explainning.